--- node_modules/angular-material/angular-material.js	2016-10-03 13:26:43.726909008 +0200
+++ node_modules/angular-material/angular-material.gislab.js	2016-10-03 13:28:23.782913781 +0200
@@ -32241,11 +32241,11 @@
     elements.wrapper = node.querySelector('md-tabs-wrapper');
     elements.canvas  = elements.wrapper.querySelector('md-tabs-canvas');
     elements.paging  = elements.canvas.querySelector('md-pagination-wrapper');
     elements.inkBar  = elements.paging.querySelector('md-ink-bar');
 
-    elements.contents = node.querySelectorAll('md-tabs-content-wrapper > md-tab-content');
+    elements.contents = node.querySelectorAll(':scope > md-tabs-content-wrapper > md-tab-content');
     elements.tabs    = elements.paging.querySelectorAll('md-tab-item');
     elements.dummies = elements.canvas.querySelectorAll('md-dummy-tab');
 
     return elements;
   }
